### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by talentlessguy**
This PR re-formats the readme to be more readable. Also "Install" section is added for quick copypaste

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,16 @@
 # 🖇 Hooks
 
-Commonly used Ethereum hooks:
+Commonly used Ethereum hooks.
 
-`useOnBlock(provider, fn)`
+## Install
+
+```sh
+npm install eth-hooks
```
Comment (ShravanSunder) [inline README.md:8]: this should be yarn, its what we use for eth hooks and scaffold eth

**Review by ShravanSunder — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (talentlessguy) [reply README.md]: @ShravanSunder oops my bad, fixed this now

**Comment by talentlessguy** [comment]
is everything good with the PR?

**Comment by ShravanSunder** [comment]
Yes!  I will approve

**Pull request merged**